src/pat/auto-suggest/auto-suggest.js

```diff
@@ -11,7 +11,10 @@ const log = logging.getLogger("autosuggest");
 export const parser = new Parser("autosuggest");
 parser.addArgument("ajax-data-type", "JSON");
 parser.addArgument("ajax-search-index", "");
+parser.addArgument("ajax-timeout", 400);
 parser.addArgument("ajax-url", "");
+parser.addArgument("max-initial-size", 10); // AJAX search results limit for the first page.
+parser.addArgument("ajax-batch-size", 10); // AJAX search results limit for subsequent pages.
```

Comment:
In https://github.com/Patternslib/Patterns/pull/1195#issuecomment-2208705609 it says that the default for `ajax-batch-size` is supposed to be 0, in which case IIUC no batching is performed. Is there a reason that it is done differently here?

Reply:
fixed!